src/components/HomeBlocks/LeaderboardsHomeBlock.tsx

```diff
@@ -64,37 +80,46 @@ export const LeaderboardsHomeBlock = ({ homeBlock }: Props) => {
           },
         }}
       >
-        {leaderboards.map((leaderboard) => (
-          <Carousel.Slide key={leaderboard.id}>
-            <Card>
-              <Group position="apart" align="center">
-                <Text size="lg">{leaderboard.title}</Text>
-                <Link href={`/leaderboard/${leaderboard.id}`} passHref>
-                  <Button
-                    rightIcon={<IconArrowRight size={16} />}
-                    variant="subtle"
-                    size="xs"
-                    compact
-                  >
-                    More
-                  </Button>
-                </Link>
-              </Group>
-              <Stack mt="md">
-                {leaderboard.results.map((result, idx) => {
-                  const isLastItem = idx === leaderboard.results.length - 1;
+        {leaderboards.map((leaderboard) => {
+          const displayedResults = leaderboard.results.slice(0, 4);
+
+          return (
+            <Carousel.Slide key={leaderboard.id}>
+              <Card sx={{ minHeight: '100%' }}>
+                <Group position="apart" align="center">
+                  <Text size="lg">{leaderboard.title}</Text>
+                  <Link href={`/leaderboard/${leaderboard.id}`} passHref>
+                    <Button
+                      rightIcon={<IconArrowRight size={16} />}
+                      variant="subtle"
+                      size="xs"
+                      compact
+                    >
+                      More
+                    </Button>
+                  </Link>
+                </Group>
+                <Stack mt="md">
+                  {displayedResults.length === 0 && (
+                    <Text sx={{ opacity: 0.5 }}>
```

Comment:
Maybe do `color="dimmed"` to keep consistency? If that doesn't match the designs, ignore what I just mentioned 😬 
```suggestion
                    <Text color="dimmed">
```